vms/registration/views.py

```diff
@@ -186,11 +193,22 @@ def post(self, request):
 
                     volunteer.reminder_days = 1
                     volunteer.save()
-                    registered = True
-
-                    messages.success(request,
-                                     'You have successfully registered!')
-                    return HttpResponseRedirect(reverse('home:index'))
+                    user.is_active = False
+                    current_site = get_current_site(request)
+                    mail_subject = 'Activate your blog account.'
+                    message = render_to_string(
+                        'registration/acc_active_email.html', {
+                            'user': user,
+                            'domain': current_site.domain,
+                            'uid': urlsafe_base64_encode(force_bytes(user.pk)),
+                            'token': account_activation_token.make_token(user),
+                        })
+                    to_email = volunteer_form.cleaned_data.get('email')
+                    email = EmailMessage(mail_subject, message, to=[to_email])
+                    email.send()
+                    return HttpResponse(
+                        'Please confirm your email address to complete the registration'
+                    )
                 else:
                     print(user_form.errors, volunteer_form.errors)
```

Comment:
I am just wondering why we have a print() here :-1:

Reply:
@tonythomas01 I have not added this print statement. Its almost there. Do you want me to remove it?